specification/azuredatatransfer/resource-manager/Microsoft.AzureDataTransfer/preview/2023-10-11-preview/azuredatatransfer.json

```diff
@@ -0,0 +1,2304 @@
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "actionRequest": {
+      "type": "object",
+      "description": "An action request",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the action"
+        },
+        "payload": {
+          "type": "object",
+          "description": "Action request payload",
+          "additionalProperties": {}
+        }
+      }
+    },
+    "actionResult": {
+      "type": "object",
+      "description": "An action result",
+      "properties": {
+        "result": {
+          "type": "object",
+          "description": "Action request result",
+          "additionalProperties": {}
+        }
+      }
+    },
+    "selectedResource": {
+      "type": "object",
+      "description": "A resource selected from ARM",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the connection"
+        },
+        "id": {
+          "type": "string",
+          "description": "Id of the connection"
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the connection",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "subscriptionName": {
+          "type": "string",
+          "description": "Name of the subscription with the connection"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "flowProperties": {
+      "type": "object",
+      "properties": {
+        "connection": {
+          "$ref": "#/definitions/selectedResource",
+          "description": "The connection associated with this flow"
+        },
+        "flowId": {
+          "type": "string",
+          "description": "Dataflow GUID associated with this flow",
+          "readOnly": true
+        },
+        "keyVaultUri": {
+          "type": "string",
+          "description": "AME, PME, or TORUS only! AKV Chain Containing SAS Token"
+        },
+        "linkStatus": {
+          "enum": [
+            "Linked",
+            "Unlinked"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "linkStatusFlow",
+            "modelAsString": true
+          },
+          "description": "Link status of the current flow",
+          "readOnly": true
+        },
+        "linkedFlowId": {
+          "type": "string",
+          "description": "Resource ID of the linked flow",
```

Comment:
>description [](http://example.com/codeflow?start=11&length=11)

"format": "arm-id"?

Reply:
The resource is not guaranteed to be in the same tenant, so I believe we cannot use "format": "arm-id" with `x-ms-arm-id-details`.